include/solvers/file_solution_history.h

```diff
@@ -97,7 +97,7 @@ class FileSolutionHistory : public SolutionHistory
   stored_solutions_iterator stored_sols;
 
   // A helper function to locate entries at a given time
-  void find_stored_entry(Real time);
+  void find_stored_entry(Real time, bool storing = false);
```

Comment:
We ought to document ```\p storing``` (and the reason for it) in the comment here.

Reply:
Indeed, passing different values to a method's argument is expected to change its behavior.  We knew that before the redundant comment.  What's important is *how* the behavior changes.  In this case, the issues we have with floating-point comparisons and the manner in which this fixes them needs to be here for everyone to read, not in IM logs for just you and me.

Reply:
I've added a description of how the behaviour of the function changes depending on whether we are storing or not-storing.